static/blockchains/categories.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Not sure if we should add a 'gaming' category, for cases such as Terra Casino, dUST (from Depeg Nation) and others.